docs/logs/sending-container-logs.md

```diff
@@ -0,0 +1,53 @@
+title: Sending Container Logs
+description: Collect and ship docker logs 
+
+
+## Sending Container Logs
+
+[Logagent](https://sematext.com/logagent/) is a general purpose open-source log shipper. The [Logagent Docker image](https://hub.docker.com/_/logagent) is pre-configured for the [log collection on container platforms](https://sematext.com/blog/docker-container-monitoring-with-sematext/#toc-container-logs-0). It runs as a tiny container on every Docker host and collects logs for all cluster nodes and their containers. Logagent enriches container logs with container metadata such as Kubernetes, Docker Enterprise, and Docker Swarm.
+
+The deployment of Logagent is very flexible, all its [configurations options for logs](https://sematext.com/docs/logagent/installation-docker/#configuration-parameters). 
```

Comment:
The deployment of Logagent is very flexible, all its configurations options for logs => Logagent deployment is very simple.  See [all Logagent configuration options].